coreteam.py

```diff
@@ -101,8 +101,10 @@
     'ps_livetranslation': 'Live translation',
     'ps_mainmenu': 'Main menu module',
     'ps_newproducts': 'New Products module',
+    'ps_productinfo': 'Product tooltips',
     'ps_reminder': 'Reminder module',
     'ps_searchbar': 'Search Bar module',
+    'ps_searchbarjqauto': 'Search bar autocomplete',
```

Comment:
This repository should be moved.

Reply:
ok, but this is not really the right place to discuss about that. until it is moved somewhere else, there can be work on it, and it is the case with a recent merged pull request.
feel free to ping me on an issue dedicated to moving this repo and I will be glad to help to get it done.

Reply:
Agree, this repository is still in the org even if it's not used.